src/components/UpdateAlert.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react';
+import { Alert } from '@mui/material';
+
+const UpdateAlert = (): JSX.Element => {
```

Comment:
ESLint should have technically flagged this function for not having JSDocs, so can you add some? Look at some of the other component folders for examples of JSDocs (comments above each function)

Reply:
Yeah I can do that.